docs/src/Button.doc.js

```diff
@@ -9,281 +9,287 @@ import PageHeader from './components/PageHeader.js';
 const cards: Array<Node> = [];
 const card = c => cards.push(c);
 
+const buildDescription = (lines: Array<string>): Node => (
```

Comment:
@AlbertCarreras Instead of adding this to `Button.doc.js`, could we move it to `PropTable`? Basically if `description` is an array of strings, execute this logic.

Reply:
That makes more sense. Thanks

Reply:
DONE